.github/workflows/codeql.yml

```
@@ -63,9 +65,12 @@ jobs:
 
       - if: ${{ matrix.build-mode == 'manual' }}
         name: Build Templ
+        uses: actions/setup-go@v3
```

actionlint (1.7.4)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)